openapi/components/schemas/GatewayAccountConfig/Worldpay.yaml

```
@@ -19,5 +19,12 @@ allOf:
         required:
           - merchantCode
           - merchantPassword
+      settings:
+        type: object
+        description: Worldpay settings object
```

```suggestion
        description: Worldpay settings object.
```

https://rebilly.slack.com/archives/C7U0AHYJ0/p1601565453042200